docs/history/changelog-4.0.rst

```diff
@@ -0,0 +1,231 @@
+.. _changelog:
+
+================
+ Change history
+================
+
```

Comment:
But we need to create changelog.rst as well right?
Before we merge, let's make a readthedocs build to verify that this branch produces the correct results.